cmd/os_mgmt/src/os_mgmt.c

```diff
@@ -190,6 +199,68 @@ os_mgmt_taskstat_read(struct mgmt_ctxt *ctxt)
 }
 #endif
 
+/**
+ * Command handler: os daytime
+ */
+static int
+os_mgmt_datetime_read(struct mgmt_ctxt *ctxt)
+{
+    CborEncoder datetime_map;
+    CborError err;
+    char buf[33]; /* size of DATETIME_BUFSIZE */
+    err = 0;
+
+    err |= cbor_encode_text_stringz(&ctxt->encoder, "datetime");
+    if (err != 0) {
+        return MGMT_ERR_ENOMEM;
+    }
+
+    err |= os_mgmt_impl_datetime_info(buf);
+    if (err != 0) {
+        return MGMT_ERR_EINVAL;
+    }
+
+    err |= cbor_encode_text_stringz(&datetime_map, buf);
```

Comment:
so we write to datetime_map which is on the stack and then what we do with it ? 
Should it go to ctxt somehow or maybe it goes there magically?